spec.html

```diff
@@ -1024,6 +1024,7 @@ <h1>Mathematical Operations</h1>
       <emu-note>
         <p><emu-eqn>floor(_x_) = _x_ - (_x_ modulo 1)</emu-eqn>.</p>
       </emu-note>
+      <p>An <dfn id="interval">interval</dfn> from lower bound _a_ to upper bound _b_ is a possibly-infinite, possibly-empty set of numeric values of the same numeric type. A numeric value _x_ is included in the interval if it satisfies both its <em>lower bound condition</em> and its <em>upper bound condition</em>. If an interval's lower bound is described as inclusive, its lower bound condition is _a_ &le; _x_; otherwise, it's lower bound is exclusive and its lower bound condition is _a_ &lt; _x_. If an interval's upper bound is described as inclusive, its upper bound condition is _x_ &le; _b_; otherwise, it's upper bound is exclusive and its upper bound condition is _x_ &lt; _b_. For example, the interval from 1 (inclusive) to 2 (exclusive) consists of all mathematical values between 1 and 2, including 1 and not including 2. An <dfn id="inclusive-interval">inclusive interval</dfn> is an interval where both bounds are inclusive. It is an editorial error if any bound of an interval is not described as either inclusive or exclusive. For the purpose of defining intervals, *-0*<sub>𝔽</sub> &lt; <!-- this comment here to avoid ecmarkup complaining about comparison to 0 --> *+0*<sub>𝔽</sub>, so, for example, an inclusive interval with a lower bound of *+0*<sub>𝔽</sub> includes *+0*<sub>𝔽</sub> but not *-0*<sub>𝔽</sub>. *NaN* is never included in an interval.</p>
```

Comment:
```suggestion
      <p>An <dfn id="interval">interval</dfn> from lower bound _a_ to upper bound _b_ is a possibly-infinite, possibly-empty set of numeric values of the same numeric type. A numeric value _x_ is included in the interval if it satisfies both its <em>lower bound condition</em> and its <em>upper bound condition</em>. Each bound will always be described as either inclusive or exclusive. If an interval's lower bound is described as inclusive, its lower bound condition is _a_ &le; _x_; if its lower bound is described as exclusive, its lower bound condition is _a_ &lt; _x_. If an interval's upper bound is described as inclusive, its upper bound condition is _x_ &le; _b_; if its upper bound is described as exclusive, its upper bound condition is _x_ &lt; _b_. For example, the interval from 1 (inclusive) to 2 (exclusive) consists of all mathematical values between 1 and 2, including 1 and not including 2. An <dfn id="inclusive-interval">inclusive interval</dfn> is an interval where both bounds are inclusive. It is an editorial error if any bound of an interval is not described as either inclusive or exclusive. For the purpose of defining intervals, *-0*<sub>𝔽</sub> &lt; <!-- this comment here to avoid ecmarkup complaining about comparison to 0 --> *+0*<sub>𝔽</sub>, so, for example, an inclusive interval with a lower bound of *+0*<sub>𝔽</sub> includes *+0*<sub>𝔽</sub> but not *-0*<sub>𝔽</sub>. *NaN* is never included in an interval.</p>
```

Reply:
Adopted with tweaks.

Reply:
Works for me, thanks.